XenModel/XenAPI/HTTP.cs

```diff
@@ -38,6 +38,7 @@
 using System.Security.Authentication;
 using System.Security.Cryptography.X509Certificates;
 using System.Runtime.Serialization;
+using XenAdmin.Core;

```

Comment:
This file/class comes from the C# SDK, which knows nothing about XenCenter, hence we cannot use references to XC namespaces. If there are methods useful for the SDK, consider moving them to that level, otherwise consider moving the code using them from this class to a XC only class extension.